src/Interactive/Features/Interactive/Core/NuGetPackageResolverImpl.cs

```diff
@@ -64,9 +64,16 @@ internal override ImmutableArray<string> ResolveNuGetPackage(string packageName,
                     var configPath = Path.Combine(tempPath, "nuget.config");
                     File.WriteAllText(configPath, EmptyNuGetConfig);
 
+                    // Copy nuget.exe resource to temp directory. (nuget.exe is an
+                    // implementation detail here and embedding nuget.exe as a
+                    // resource rather than installing nuget.exe as a separate exe
+                    // means we won't add nuget.exe unnecessarily to the path.)
+                    var nugetExePath = Path.Combine(tempPath, "nuget.exe");
+                    File.WriteAllBytes(nugetExePath, Resources.NuGetExe);
```

Comment:
It might be nice to keep nugetExePath as a static and not read/write it for each package that's resolved...but then again, maybe that's small potatoes compared to everything else going on here...